config/jobs/kubernetes-security/genjobs.go

```diff
@@ -188,7 +188,7 @@ func convertJobToSecurityJob(j *config.Presubmit, dropLabels sets.String, jobsCo
 				endsWithScenarioArgs = true
 
 				// handle --repo substitution for main repo
-			} else if strings.HasPrefix(arg, "--repo=k8s.io/kubernetes") || strings.HasPrefix(arg, "--repo=k8s.io/$(REPO_NAME)") {
+			} else if (strings.HasPrefix(arg, "--repo=k8s.io/kubernetes") || strings.HasPrefix(arg, "--repo=k8s.io/$(REPO_NAME)")) && !strings.Contains(arg, "kubernetes-anywhere") {
```

Comment:
handle `kubernetes-anywhere`.
there could be better ways to do this.

Reply:
perhaps:
`} else if (arg == "--repo=k8s.io/kubernetes" || strings.HasPrefix(arg, "--repo=k8s.io/kubernetes=") || arg == "--repo=k8s.io/$(REPO_NAME)" || strings.HasPrefix(arg, "--repo=k8s.io/$(REPO_NAME)=")) {`

the repo flag allows:
- just the repo name, no qualifiers
- the repo name, and then `=` with some qualifier on branch, sha etc.
- may contain env, for a presubmit in k/k (which is all we generate here, except k-s/k), the `$(REPO_NAME)` is kubernetes

Reply:
i've amended the commit with the `else if` change.
it generated the same config, so seems to work correctly.

Reply:
thanks!